core/src/main/resources/db/migration.sql

```diff
@@ -9,3 +9,9 @@ CREATE TABLE sample_list_list LIKE patient_list_list;
 INSERT sample_list_list SELECT * FROM patient_list_list;
 ALTER TABLE sample_list_list CHANGE PATIENT_ID SAMPLE_ID INT(11);
 UPDATE info SET DB_SCHEMA_VERSION="1.1.0";
+
+##version: 1.2.0
+ALTER TABLE cna_event AUTO_INCREMENT=1;
+UPDATE info SET DB_SCHEMA_VERSION="1.2.0";
+ALTER TABLE mutation add UNIQUE KEY `UQ_MUTATION_EVENT_ID_GENETIC_PROFILE_ID_SAMPLE_ID` (`MUTATION_EVENT_ID`,`GENETIC_PROFILE_ID`,`SAMPLE_ID`) COMMENT 'Constraint to block duplicated mutation entries.';
```

Comment:
Having a problem running this command.  Is a COMMENT only valid when a UNIQUE INDEX is created (I can get this to run by dropping COMMENT component.